front/src/styles/scss/_overrideBootstrapSNCF.scss

```diff
@@ -147,3 +147,12 @@ input {
 .input-group-sm input {
   padding-bottom: 0.45rem;
 }
+
+/*
+*
+*   Z-INDEX
+*
+*/
+.zindex-dropdown {
+  z-index: 1000 !important;
```

Comment:
Why 1000? Please choose a reasonable value :D

Reply:
It's the bootstrap default value for dropdown z-index
https://getbootstrap.com/docs/4.6/layout/overview/#z-index

Reply:
Yes, and it's already in SNCF's bootstrap css code.
You've got a `position` problem, not a z-index problem, it's because of `position-static` class.